webapp/src/components/AssessmentMetadataBar.tsx

```diff
@@ -0,0 +1,260 @@
+import React, { useState, useRef, useEffect } from 'react';
+
+import {
+  Avatar,
+  Divider,
+  List,
+  ListItem,
+  ListItemAvatar,
+  ListItemText,
+  Tooltip,
+  Typography,
+  Switch,
+} from '@mui/material';
+import InfoIcon from '@mui/icons-material/Info';
+import InventoryIcon from '@mui/icons-material/Inventory';
+import TimerIcon from '@mui/icons-material/Timer';
+import CalendarMonthIcon from '@mui/icons-material/CalendarMonth';
+import CheckCircleIcon from '@mui/icons-material/CheckCircle';
+import LightbulbIcon from '@mui/icons-material/Lightbulb';
+import { blue, grey } from '@mui/material/colors';
+
+import {
+  Assessment,
+  SubmissionStatus,
+  AssessmentType,
+  AssessmentSubmission,
+} from '../assets/data';
+import { formatDateTime } from '../helpers/dateTime';
+
+interface AssessmentMetadataBarProps {
+  assessment: Assessment;
+  submission: AssessmentSubmission;
+}
+
+const AssessmentMetadataBar = ({
+  assessment,
+  submission,
+}: AssessmentMetadataBarProps) => {
+  const dueTime = new Date(assessment?.dueDate!);
+  //TODO: use the opened date from the submission
+  const [openedTime] = useState(new Date());
```

Comment:
Do we need another variable for this? Shouldn't it be stored inside `submission`?